EnglishFilter/sections/foreign.txt

```diff
@@ -2155,6 +2155,8 @@ mob.hr##.side_banner
 ! https://github.com/AdguardTeam/AdguardFilters/issues/50421
 ||imedia.cz^$domain=televizeseznam.cz
 ||i.imedia.cz/json|$replace=/(<VAST[\s\S]*?>)[\s\S]*<\/VAST>/\$1<\/VAST>/,domain=televizeseznam.cz
+! https://github.com/AdguardTeam/AdguardFilters/issues/51143
+lingea.sk#?#.container > div.col-sm-9 > div.bx-wrapper:has(> div.bx-viewport > div#promoCarousel)
```

Comment:
I don't see it.

Reply:
<details>
<summary>Screenshot</summary>
<img src="https://user-images.githubusercontent.com/18536328/76543157-23b31680-6497-11ea-81a1-96c3be27304a.png"/>
</details>